src/index.html

```diff
@@ -13,10 +13,16 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/blocks/index.scss"
```

Comment:
The stylesheet link is pointing to an SCSS file (`index.scss`). Browsers cannot directly interpret SCSS files; they need to be compiled into CSS. Ensure that the SCSS is compiled to CSS and link the resulting CSS file instead.